package-lock.json

```diff
@@ -4656,6 +4656,13 @@
       "integrity": "sha512-V9znK0zc3jOPV4VD2zZn0sDhZU3WAE2bmlxdIwwQPPzPjvyLkd8B3JUVdS1IDUFDkWZ72c9qnv1GK2ZagTZ8EA==",
       "requires": {
         "d3-color": "1"
+      },
+      "dependencies": {
+        "d3-color": {
+          "version": "1.4.1",
```

Comment:
It looks like we're not using `d3-color` directly, and not all versions of it are being updated. I think we should be upgrading our direct dependencies instead.